programs/vote/src/vote_state/mod.rs

```diff
@@ -1778,6 +1830,307 @@ mod tests {
         }
     }
 
+    // Test vote credit updates after "timely vote credits" feature is enabled
+    #[test]
+    fn test_timely_credits() {
+        // Each of the following (Vec<Slot>, Slot, u32) tuples gives a set of slots to cast votes on, a slot in which
+        // the vote was cast, and the number of credits that should have been earned by the vote account after this
+        // and all prior votes were cast.
+        let test_vote_groups: Vec<(Vec<Slot>, Slot, u32)> = vec![
+            // Initial set of votes that don't dequeue any slots, so no credits earned
+            (
+                vec![1, 2, 3, 4, 5, 6, 7, 8],
+                9,
+                // root: none, no credits earned
+                0,
+            ),
```

Comment:
Initializing a struct with named fields instead of a tuple here would be easier to read, especially in the logic that comes later when we are referencing these fields

Reply:
I've updated with comments; unless you're very insistent on changing the tabular form of this data, I'm inclined to leave it as is. Please let me know, thanks!